modules/virt-example-enabling-lldp-policy.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-example-enabling-lldp-policy_{context}"]
+= Example: Node network configuration policy to enable LLDP reporting
+
+The following YAML file is an example of a `NodeNetworkConfigurationPolicy` manifest that enables the Link Layer Discovery Protocol (LLDP) listener for all ethernet ports in your {product-title} cluster. Devices on a local area network can use LLDP to advertise their identity, capabilities, and neighbor information.
```

Comment:
I think we have to enable LLDP first, right?

```
cat | sudo nmstatectl apply - << EOL
interfaces:
  - name: eth1
    type: ethernet
    lldp:
      enabled: true
EOL
```

LLDP has RX and TX and RXTX modes as well, so we want RXTX.

Reply:
NM only supports LLDP RX?  That's not much fun.  :disappointed:

Reply:
The policy failed to apply
```
NmstateError: VerificationError: Verification failure: enx30d0425666bc.interface.profile-name desire '"ethernet-enx30d0425666bc-10773"', current '"Wired Connection"'
```

It sounds like the policy is correct, but I just hit a different `nmstate` issue?

Reply:
Yes totally different issue, feels to me that the predefined NetworkManager profile is colliding with the one generated by nmstate
@cathay4t do this rings any bell ?

Reply:
No idea. Please report a issue to nmstate, we will track it there.